stl/inc/yvals_core.h

```diff
@@ -35,6 +35,7 @@
 // P0646R1 list/forward_list remove()/remove_if()/unique() Return size_type
 // P0653R2 to_address()
 // P0655R1 visit<R>()
+// P0738R2 istream_iterator Cleanup. The minstrel know it as "I Stream, You Stream, We All Stream for istream_iterator"

```

Comment:
This line should be removed, leaving only the entry in "implemented unconditionally".

Reply:
:sadface: